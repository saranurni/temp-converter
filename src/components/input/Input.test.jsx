import renderer from "react-test-renderer";
import Input from "./Input";

it("renders correctly", () => {
  const tree = renderer.create(
    <Input label="" value={0} changeHandler={() => {}} />
  );

  expect(tree).toMatchSnapshot();
});
