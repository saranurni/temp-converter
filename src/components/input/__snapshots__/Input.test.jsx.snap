// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col gap-y-1"
>
  <label
    htmlFor=""
  />
  <input
    className="w-36 rounded border"
    id=""
    onChange={[Function]}
    type="number"
    value={0}
  />
</div>
`;
